Mathlib/Analysis/LocallyConvex/AbsConvex.lean

```diff
@@ -40,6 +42,95 @@ open NNReal Pointwise Topology
 
 variable {𝕜 E F G ι : Type*}
 
+section AbsolutelyConvex
+
+variable (𝕜) [SeminormedRing 𝕜] [SMul 𝕜 E] [SMul ℝ E] [AddCommMonoid E]
+/-- The type of absolutely convex sets. -/
+def AbsConvex (s : Set E) : Prop := Balanced 𝕜 s ∧ Convex ℝ s
+
+variable {𝕜}
+
+theorem absConvex_empty : AbsConvex 𝕜 (∅ : Set E) := ⟨balanced_empty, convex_empty⟩
```

Comment:
```suggestion
theorem AbsConvex.empty : AbsConvex 𝕜 (∅ : Set E) := ⟨balanced_empty, convex_empty⟩
```
for dot notation. Same below. Feel free to open PRs to align the `Balanced` and `Convex` API to that new convention, btw